src/bin/rust-ar.rs

```diff
@@ -1,10 +1,5 @@
-extern crate cargo_binutils as cbu;
-
-use std::process;
+extern crate cargo_binutils as binutils;
```

Comment:
Those `extern crate` look a bit old school. I'm also not convinced that the rewrite of `cargo_binutils` as `binutils` is much of a benefit. 😅